.github/workflows/xk6-tests/xk6-output-test/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/grafana/xk6-output-test
 go 1.19
 
 require (
-	github.com/spf13/afero v1.9.2
-	go.k6.io/k6 v0.46.0
+	github.com/spf13/afero v1.1.2
```

Comment:
Also downgraded afero here to the vesion k6 uses

Reply:
We might put the `modtools_frozen.yml` or a symlink here